### ISSUE TITLE
I need help with something

### ISSUE THREAD

**Issue by Ant9cp** [Monika-After-Story/MonikaModDev#7751]
Do i need to type in .txt after the oki doki file? for it to work? for monika's birthday party because i think i'm doing something wrong because i'm not getting a response.

**Comment by toonziz** [comment]
no, remove the txt. (not the file, the little .txt) then, open the game, and close it, the file name should now say "gotcha". 

also why r so many ppl freaking out abt monika's birthday? its on the 22nd, its the 15th rn.

**Comment by Ant9cp** [comment]
Um it isn't your birthday everyday you know, only once a year

but thanks for the help

**Comment by Ant9cp** [comment]
Both files one that's addresed to you and the oki doki file both disappeared 
i don't have a file that says gotcha 


well i do but that was from last years birthday

**Comment by toonziz** [comment]
> Both files one that's addresed to you and the oki doki file both disappeared
> i don't have a file that says gotcha
> 
> well i do but that was from last years birthday

[untitled folder 4.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7174371/untitled.folder.4.zip)



here. it has the gotcha file in there, btw i saw the comment u deleted.

**Comment by annonymous-user123** [comment]
So, today is Monika's Birthday in my time zone, so what now?

**Comment by annonymous-user123** [comment]
I mean, I have the gotcha file, but I opened the game and..... Its the same. Do I move Monika's file?

**Comment by Booplicate** [comment]
Open a new issue/discussion if you have a problem. You don't have to move the file, you just need to confirm the party and use the `take you somewhere` farewell.
